packages/config/config/devices/0x0258/Motion Sensor 3

```diff
@@ -0,0 +1,174 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "Motion Sensor 3",
```

Comment:
```suggestion
	"label": "NAS-PD01Z",
```
Mentioning the "revision" in the description is okay.

Reply:
Happy to go with this model number - But I believe Neo Coolcam have used that for multiple versions of this.